src/windows-emulator/windows_emulator.cpp

```diff
@@ -220,6 +220,10 @@ void windows_emulator::setup_process(const application_settings& app_settings, c
     const auto& emu = this->emu();
     auto& context = this->process;
 
+    mod_manager.on_module_load = std::move(callbacks.module_loaded);
+    mod_manager.on_module_unload = std::move(callbacks.module_unloaded);
+    context.on_create_thread = std::move(callbacks.thread_created);
+
```

Comment:
this should be done in the constructor. setup process may be called multiple times (e.g. in the unit test), which results in moving an already moved object and overwriting the values.